src/test/java/org/jenkinsci/plugins/workflow/steps/CoreStepTest.java

```diff
@@ -45,9 +48,11 @@
 
     @ClassRule public static BuildWatcher buildWatcher = new BuildWatcher();
     @Rule public JenkinsRule r = new JenkinsRule();
+    private SnippetizerTester st = new SnippetizerTester(r);
 
     @Test public void artifactArchiver() throws Exception {
         WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        // TODO 2.7.x+ use core symbols
         p.setDefinition(new CpsFlowDefinition("node {writeFile text: '', file: 'x.txt'; step([$class: 'ArtifactArchiver', artifacts: 'x.txt', fingerprint: true])}", true));
```

Comment:
Ah, that answers my question from https://github.com/jenkinsci/workflow-cps-plugin/pull/28. =)